analyses/doublet-detection/benchmark-test-data/generate-benchmark-test-data.R

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env Rscript
+
+# This script generates a test data file for use when testing the benchmarking script `../run_doublet-detection-benchmark.sh`.
+# The test data file is a subsetted version of the `hm-6k` dataset used for benchmarking, as described in `../README.md`.
+# This script assumes that ../scratch/benchmark-datasets/raw contains the .rds files from <https://doi.org/10.5281/zenodo.4562782>'s `real_datasets.zip` file.
+
+
+library(Matrix)
+
+input_file <- file.path("..", "scratch", "benchmark-datasets", "raw", "hm-6k.rds")
+output_file <- file.path("hm-6k_subset.rds")
```

Comment:
These paths should be set relative to something concrete and not assume that this script will be run from its current location.

Reply:
I added `projroot::find_root(rprojroot::is_renv_project)` here - https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/624/commits/59ec24970992aa440e6b1505c31b300c59cb032b